internal/action/install.go

```diff
@@ -55,6 +55,11 @@ func Install(ctx context.Context, config *helmaction.Configuration, obj *v2.Helm
 	if err != nil {
 		return nil, err
 	}
+
+	if err := helmchartutil.ProcessDependenciesWithMerge(chrt, vals); err != nil {
+		return nil, err
+	}
```

Comment:
If this actually works, we'll need it on upgrade too.

Reply:
That makes sense. I should probably add it to `applyCRDs` instead to cover all the use-cases